src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,115 @@
+
+.cards {
+  display: flex;
+  box-sizing: border-box;
+  flex-wrap: wrap;
+  width: 1024px;
+  margin: 50px 41px 73px;
```

Comment:
cards not centered
```suggestion
  margin: 0 auto;
  padding: 50px 40px
```
![Screenshot from 2023-05-29 12-02-32](https://github.com/mate-academy/layout_catalog/assets/33029864/987e7759-00bb-4737-8593-aff611100a9d)